base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,56 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+    WCHAR szPath[MAX_PATH];
+    LPWSTR szExplorer;
+    BOOL bIsSystemShell = TRUE; // Assume we are the system shell by default.
```

Comment:
Since the code is C++, we can move the variables closer to where they start being used.
```suggestion
    BOOL bIsSystemShell = TRUE; // Assume we are the system shell by default.
    WCHAR szPath[MAX_PATH];
```